.github/workflows/main.yml

```diff
@@ -0,0 +1,53 @@
+name: Update README
+
+on:
+  workflow_dispatch:
+  push:
+  schedule:
+    # run at 10am UTC daily
+    - cron: '0 10 * * *'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+      - name: Install npm dependencies
+        run: npm install
+      - name: Run eslint
+        run: npm run lint
+  update:
+    if: ${{ github.event_name == 'schedule' ||
+      github.event_name == 'workflow_dispatch' }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+      - name: Install npm dependencies
+        run: npm install
+      - name: Update README
+        env:
+          GH_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: npm run update-readme
+      - name: Setup git config
+        run: |
+          # Setup the username and email. Use 'GitHub Actions Bot' with no email
+          # by default
+          git config user.name "GitHub Actions Bot"
+          git config user.email "<>"
+      - name: Commit
+        run: |
+          # Stage the file, commit and push
+          git add README.md
```

Comment:
I think this is fine for now, but it sort of tangles the purpose of this repo--it's both the updater...and the updates.

I wonder if we could/should use a `gh-pages` branch and publish both an HTML page (via Jekyll and the README--which should "just work" once you commit there and setup GH Pages on this repo) _and_ publish a JSON file of this data, so we can consume it on the dashboard.

Thoughts?

Reply:
Sounds like a good idea! Thanks for bringing this up. I've opened an issue to track this https://github.com/digitalbazaar/vc-commit-tracker/issues/2, please feel free to update the issue.